tests/destroy-data-then-repair.yaml

```diff
@@ -0,0 +1,38 @@
+test_duration: 10080
+stress_write_cmd: "cassandra-stress write cl=QUORUM n=10000 -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..300300300"
+stress_read_cmd:  "cassandra-stress read cl=ONE n=10000 -mode cql3 native -rate threads=10 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..1100200300 -port jmx=6868 -log interval=5"
+n_db_nodes: 3
+n_loaders: 4
```

Comment:
n_loaders is depended on the way we choose to load the data, in the current case there is no need for 4.
but it should change anyway.